### PR TITLE
BSDT Service field message

### PR DESCRIPTION
**Pull request by AndersonTarren**
# Critical Changes

# Changes
Adds a message at the top of BSDT letting users know that fields may change based on Service configuration.

# Issues Closed
[W-10311095](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000Xtqt1YAB/view)

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage  
- [ ] CRUD/FLS is enforced in Apex
- [ ] Permission sets are updated to account for CRUD|FLS|Tab|Classes
- [ ] Field sets are updated to account for new fields
- [x] UX approval or UX not necessary
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [x] PR contains draft release notes
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by AndersonTarren** [comment]
@jencrabtree We're looking for the right text to let a user know that based on their service configuration (fieldsets) - that the fields displayed may change when they select a service.

**Comment by jencrabtree** [comment]
@AndersonTarren on it. How will the text display? Infobubble?

**Comment by bmuniswamy** [comment]
Screen Shot 2022-07-13 at 1 27 10 PM
@jencrabtree , Here is how it will be displayed on the BSDT screen. If you need more information please let me know

File: force-app/main/default/lwc/bulkServiceDeliveryUI/bulkServiceDeliveryUI.html [hunk 1]
```diff
@@ -16,6 +16,9 @@
         </div>
 
         <lightning-card if:false={showWizard}>
+            <div class="slds-var-p-horizontal_medium">
+                {labels.serviceFieldMessage}
```
Comment (bmuniswamy) [inline force-app/main/default/lwc/bulkServiceDeliveryUI/bulkServiceDeliveryUI.html:20]: @AndersonTarren, This message is currently not read out by the screen reader. We may want to add aria-live to the div for the screen reader to read the message out.

**Review by bmuniswamy — COMMENTED** [review]
One small comment

**Comment by jencrabtree** [comment]
@AndersonTarren Approved text back from review: 

The fields shown on this page depend on the Service you select.

**Review by bmuniswamy — APPROVED** [review]
Looks good. Thanks Tarren for making the change to the text

**Pull request merged**